src/Microsoft.TemplateEngine.Authoring.TemplateVerifier/TemplateVerifierOptions.cs

```diff
@@ -88,6 +94,26 @@ public class TemplateVerifierOptions : IOptions<TemplateVerifierOptions>
         /// </summary>
         public VerifyDirectory? CustomDirectoryVerifier { get; private set; }
 
+        /// <summary>
+        /// Gets a custom prefix prepended in front of generated scenario name - result used for naming verification subdirectories.
+        /// </summary>
+        public string? ScenarioDistinguisher { get; init; }
+
+        /// <summary>
+        /// If set to true - the instantiation params are not appended to verification subdirectories.
```

Comment:
```suggestion
        /// <see langword="true" />, if the template arguments should not be appended to verification subdirectories.
```

More correct according to the guideline is:
```suggestion
        /// Gets a value that indicates if the template arguments should not be appended to verification subdirectories.
```

https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Property
But I prefer the first option :)